src/app/app.js

```diff
@@ -1,12 +1,35 @@
-import React from 'react';
-
-function App() {
-  return (
-    <div>
-      Hello world!
-    </div>
-  );
+import React, {Component} from 'react';
+import { connect } from 'react-redux';
+
+import { getPhotos } from '../actions/index.js';
```

Comment:
Don't need to include the '.js' at the end of your local module imports. The build/bundler tool is smart enough to know its javascript if you leave it pff.